docs/docs/documentation/getting-started/installation/postgres.md

```diff
@@ -5,40 +5,41 @@ PostgreSQL might be considered if you need to support many concurrent users. In
 **For Environment Variable Configuration, see** [Backend Configuration](./backend-config.md)
 
 ```yaml
----
-version: "3.7"
 services:
   mealie:
     image: ghcr.io/mealie-recipes/mealie:v1.3.2 # (3)
     container_name: mealie
+    restart: always
     ports:
         - "9925:9000" # (1)
     deploy:
       resources:
         limits:
           memory: 1000M # (2)
-    depends_on:
-      - postgres
     volumes:
       - mealie-data:/app/data/
+      - /etc/timezone:/etc/timezone:ro
     environment:
-    # Set Backend ENV Variables Here
-      - ALLOW_SIGNUP=true
-      - PUID=1000
-      - PGID=1000
-      - TZ=America/Anchorage
-      - MAX_WORKERS=1
-      - WEB_CONCURRENCY=1
-      - BASE_URL=https://mealie.yourdomain.com
+      # Set Backend ENV Variables Here
+      ALLOW_SIGNUP: true
+      PUID: 1000
+      PGID: 1000
+      MAX_WORKERS: 1
+      WEB_CONCURRENCY: 1
+      BASE_URL: http://localhost # (4)
```

Comment:
Why replace the old value? It would seem more clear to someone who doesn't know how it should be configured (no concerns re adding the note)

Reply:
The goal was to have a working compose directly from the example instead of needing tweaks, with appropriate context added to inform people of how to change it if they plan to use a domain. I did keep the original value in the note with an explanation, and figured it was sufficient.